app/grandchallenge/components/schemas.py

```diff
@@ -409,6 +409,87 @@
             "required": ["version", "type", "boxes"],
             "additionalProperties": False,
         },
+        "ELLI": {
+            "type": "object",
+            "properties": {
+                "name": {"type": "string"},
+                "type": {"enum": ["Ellipse"]},
+                "major_axis": {
+                    "type": "array",
+                    "items": {
+                        "type": "array",
+                        "items": {"type": "number"},
+                        "minItems": 3,
+                        "maxItems": 3,
+                    },
+                    "minItems": 2,
+                    "maxItems": 2,
+                },
+                "minor_axis": {
+                    "type": "array",
+                    "items": {
+                        "type": "array",
+                        "items": {"type": "number"},
+                        "minItems": 3,
+                        "maxItems": 3,
+                    },
+                    "minItems": 2,
+                    "maxItems": 2,
+                },
+                "version": {"$ref": "#/definitions/version-object"},
+                "probability": {"type": "number", "minimum": 0, "maximum": 1},
+            },
+            "required": ["version", "type", "major_axis", "minor_axis"],
+            "additionalProperties": False,
+        },
+        "ellipse-object": {
+            "type": "object",
+            "properties": {
+                "name": {"type": "string"},
+                "type": {"enum": ["Ellipse object"]},
```

Comment:
Type should be the same as the single annotation.
```suggestion
                "type": {"enum": ["Ellipse"]},
```